lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -194,16 +194,27 @@ void StagesSubject::stageChanged(UsdNotice::ObjectsChanged const& notice, UsdSta
 		// don't care about it. In those cases, the changePath will contain something like:
 		//   "/<prim>.visibility"
 		//   "/<prim>.xformOp:translate"
-		if (!changedPath.IsPrimPath())
+		if (changedPath.IsPropertyPath())
 			continue;
```

Comment:
`IsPrimPath()` checks  if a path identifies a prim but what we wanted here is to check if path identifies a property. 

calling isPrimPath() on "/" would return false and therefore subtree invalidation never occurs.